.eslintrc.js

```diff
@@ -6,7 +6,6 @@ module.exports = {
         node: true,
     },
     extends: ['airbnb/base', 'prettier'],
-    parser: 'babel-eslint',
```

Comment:
Why did you remove linter chekc?

Reply:
I added it before to create a decorator, now it's not needed as we're not using `babel`

Reply:
We don't have it on the `master` branch